docs/references/api.rst

```diff
@@ -631,6 +633,148 @@ like ``systemctl start ubuntu-advantage.service``.
         - This endpoint currently has no CLI support. Only the Python-based
           version is available.
 
+
+u.pro.attach.token.full_token_attach.v1
+============================================
+
+This endpoint allow the user to attach to a Pro subscription using
+a token.
+
+- Introduced in Ubuntu Pro Client Version: ``32~``
+- Args:
+
+  - ``token``: The token associated with a Pro subscriptionn
+  - ``auto_enable_services``: If false, the attach operation will not enable any service during the
+    operation
+
+.. tab-set::
+
+   .. tab-item:: Python API interaction
+      :sync: python
+
+      - Calling from Python code:
+
+        .. code-block:: python
+
+           from uaclient.api.u.pro.attach.token.full_token_attach.v1 import full_token_attach, FullTokenAttachOptions
+
+           options = FullTokenAttachOptions(token="TOKEN")
+           result = full_token_attach(options)
+
+      - Expected return object:
+
+        - ``uaclient.api.u.pro.attach.token.full_token_attach.v1.FullTokenAttachResult``
+
+          .. list-table::
+             :header-rows: 1
+
+             * - Field Name
+               - Type
+               - Description
+             * - ``enabled``
+               - *List[str]*
+               - The services enabled during the attach operation
+             * - ``reboot_required``
+               - *bool*
+               - If the system requires a reboot after the attach operation
```

Comment:
Presumably this is True if the system requires a reboot?

Reply:
Yes, do you think we should make this explicit ?

Reply:
Yes, I will make it more explicit